pkg/storage/tpr/install_types.go

```diff
@@ -45,20 +61,48 @@ func InstallTypes(cs clientset.Interface) error {
 		}
 
 		glog.Infof("Creating Third Party Resource Type: %s", tpr.Name)
-		if _, err := tprs.Create(&tpr); err != nil {
-			glog.Errorf("Failed to create Third Party Resource Type: %s (%s))", tpr.Name, err)
+		errChan = make(chan error, 1)
+		defer close(errChan)
+		errChans[tpr.Name] = errChan
```

Comment:
@MHBauer Using a WaitGroup and a timeout might cause the func exit prematurely and hide the real error returns by the client. I would rather let the http client handles the timeout.